cms/djangoapps/contentstore/tests/test_orphan.py

```diff
@@ -79,10 +173,13 @@ def test_mongo_orphan(self):
         location = self.course.location.replace(category='html', name='OrphanHtml')
         self.assertIn(location.to_deprecated_string(), orphans)
 
-    def test_mongo_orphan_delete(self):
+    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_mongo_orphan_delete(self, default_store):
```

Comment:
The tests here are definitely in need of a clean up—I tagged you here just to make sure this was moving in the right direction.